tests/ui/perfherder/graphs_view_test.jsx

```diff
@@ -231,6 +232,41 @@ test('Using select query param displays tooltip for correct datapoint', async ()
   expect(revision).toBeInTheDocument();
 });
 
+test('Test graph tooltip information is according to test data', async () => {
```

Comment:
This is a good attempt but this test seems like overkill. I think we need to test for what would be the most prone to errors.

The test before it on line 216 is already testing that the tooltip revision is what we'd expect based on the `selectedDataPoint` query parameters. It can be extended (in the same test) by asserting on the repository since `revision` and `repository` are unique together which would be the equivalent to the `dataPointId` (the unique `id` in the mock `testData`). 

I encourage you to look at the GraphTooltip code and how it's used in the GraphsContainer. Also look at this code in use [here](https://treeherder.mozilla.org/perf.html#/graphs?highlightAlerts=1&highlightedRevisions=5e69563343eb5&selected=1646990,1061068230&series=mozilla-central,1646990,1,1&series=mozilla-central,1647462,1,1&timerange=1209600). It shows two different tests run on the same framework (talos), repository (mozilla central), and platform (linux64). The vertical line makes it easier to see that there are two different tests for the same commit/push on the graph.

So if you also wanted to test that this tooltip is showing the correct revision/commit for the test you expect (a good idea), you would need to add more data to our mock `testData` file. You can do a copy paste and give it new signature/hash properties (they can be fake). Then assert that the signature_id is what is expected.

Reply:
>Also look at this code in use here

You just forgot to add the hyperlink to ```this```, causing troubles for me.
Could you please link it

>you would need to add more data to our mock testData file. You can do a copy paste and give it new signature/hash properties

The testData in the graph_view_test.js does [contain signature_id already ](https://github.com/mozilla/treeherder/blob/master/tests/ui/mock/performance_summary.json#L5). Do you mean to use that. 


In my next commit, I have removed the test_perfherder file(selenium test), commenting my test of ```Test graph tooltip information is according to test data```
I have also tested the repository name in ``` Using select query param displays tooltip for correct datapoint```
Please tell me what I need to do next. Or I will delete the new ```Test graph tooltip information is according to test data``` in my next commit

Reply:
@ionutgoldan @sarah-clements Could you please review this. 
The travis is failing due to linting errors .
I will correct them if the other things are right

Reply:
I included the hyperlink in the word 'here' - at the end of "Also look at this code in use here".

https://treeherder.mozilla.org/perf.html#/graphs?highlightAlerts=1&highlightedRevisions=5e69563343eb5&selected=1646990,1061068230&series=mozilla-central,1646990,1,1&series=mozilla-central,1647462,1,1&timerange=2592000

The linting errors should be corrected as you work - please follow the suggestions in our docs: https://treeherder.readthedocs.io/testing.html#validating-javascript